chapters/package-information.md

```diff
@@ -1417,6 +1417,54 @@ EXAMPLE 2 RDF: Property `spdx:attributionText` in class `spdx:Package`
 </Package>
 ```
 
+## 7.24 Package Purpose field <a name="7.24"></a>
+
+### 7.24.1 Description
+
+This field provides information about the purpose of the identified package. Package Purpose is intrinsic to how the package is being used rather than the content of the package. The options to populated this field are limited to:
+
+`APPLICATION` if the package is a software application;
+`FRAMEWORK` if the package is a software framework;
+`LIBRARY` if the package is a software library;
+`CONTAINER` if the package refers to a container image which can be used by a container runtime application;
+`OPERATING-SYSTEM` if the package refers to an operating system;
+`DEVICE` if the package refers to a chipset, processor, or electronic board;
+`FIRMWARE` if the package provides low level control over a device's hardware;
+`SOURCE` if the package is a collection of source files;
+`ARCHIVE` if the package refers to an archived collection of files (.tar, .zip, etc);
+`FILE` if the package is a single file which can be independently distributed (configuration file, statically linked binary, Kubernetes deployment, etc);
+`OTHER` if the package doesn't fit into the above categories.
+
+The metadata for the Package Purpose field is shown in Table 36.
+
+**Table 36 — Metadata for the package name field**
+
+| Attribute | Value |
+| --------- | ----- |
+| Required | Yes |
+| Cardinality | 0..* |
```

Comment:
If Cardinality will be `0..*`, then I think Required should be "No" rather than "Yes"?